code/game/objects/items/rogueitems/needle.dm

```diff
@@ -70,6 +70,8 @@
 
 /obj/item/needle/attack_obj(obj/O, mob/living/user)
 	var/obj/item/I = O
+	if(isnull(I) || istype(I, /obj/item))
```

Comment:
```suggestion
/obj/item/needle/attack_obj(obj/item/target, mob/living/user)
	if(!istype(target))
```

Reply:
Nevermind- after discussion on Discord it's probably better to keep the first argument inline w/ the parent proc